app/models/timeline.js

```diff
@@ -203,6 +203,8 @@ export function addModel(dbAdapter) {
       return t2 - t1
     })
 
+    posts = posts.slice(offset, offset + limit)
```

Comment:
So for the fifth page it will retrieve 150 posts and cut off the first 120? It looks... wasteful?